schema-loader/src/test/java/com/scalar/db/schemaloader/SchemaLoaderTest.java

```diff
@@ -916,6 +916,616 @@ public void tearDown() throws Exception {
     }
   }
 
+  // For backward compatibility with the old method signatures.
```

Comment:
Ditto.

Reply:
Addressed it in https://github.com/scalar-labs/scalardb/pull/2760/commits/e7b6082f33b1a769d760023ebc5b4b31c1be4456